diff --git a/dist/server/utils/requestResponseUtils.js b/dist/server/utils/requestResponseUtils.js
index a4182fa0fce700a59d093b1ab9e3c6f02fe6ee4e..4db971ec779827639245b68cd013bec0c670fb85 100644
--- a/dist/server/utils/requestResponseUtils.js
+++ b/dist/server/utils/requestResponseUtils.js
@@ -9,6 +9,7 @@ function setPrivateAuthResultOnRequest({ req, authResult }) {
 exports.setPrivateAuthResultOnRequest = setPrivateAuthResultOnRequest;
 // Tries to extract auth result from the request using several strategies
 function getAuthResultFromRequest(req) {
+    console.log(getPrivateAuthResult(req), getHeader(req, types_1.AUTH_RESULT), getQueryParam(req, types_1.AUTH_RESULT))
     return getPrivateAuthResult(req) || getHeader(req, types_1.AUTH_RESULT) || getQueryParam(req, types_1.AUTH_RESULT);
 }
 exports.getAuthResultFromRequest = getAuthResultFromRequest;