diff --git a/build/esm/hooks/field/reducer.mjs b/build/esm/hooks/field/reducer.mjs
index 6fb207c81eb61ea59add0beb84941ff12412e60c..e5a388a339f7e6156bdfeecc4ea306a03394f96a 100644
--- a/build/esm/hooks/field/reducer.mjs
+++ b/build/esm/hooks/field/reducer.mjs
@@ -102,7 +102,7 @@ function useFieldReducer(value, dirtyStateComparator) {
 }
 function initialFieldState(value) {
   return {
-    value,
+    value: value,
     defaultValue: value,
     error: undefined,
     touched: false,