diff --git a/build/lib/index.mjs b/build/lib/index.mjs
index 5dcb3188c7eb8ea7ba97dbbf6413071635cb430d..1f003b9cbbccc1f0c6b5ce3ba8912c8784379ffd 100644
--- a/build/lib/index.mjs
+++ b/build/lib/index.mjs
@@ -1,4 +1,4 @@
-import './setBatchUpdatesFn.mjs';
+// import './setBatchUpdatesFn.mjs';
 export * from '@tanstack/query-core';
 export { useQueries } from './useQueries.mjs';
 export { useQuery } from './useQuery.mjs';